meta-ostro-xt/recipes-devtools/checksec/checksec_1.5-1.bbappend

```diff
@@ -0,0 +1,3 @@
+SRC_URI[md5sum] = "075996be339ab16ad7b94d6de3ee07bd"
+SRC_URI[sha256sum] = "77b8a7fd9393d10def665658a41176ee745d5c7969a4a0f43cefcc8a4cd90947"
```

Comment:
@mythi, With this PR ostro-os-xt is using checksec_1.5-1.bb which is coming from ostro-os only.

I tried re-build checksec  without this patch by disabling source mirror(setting OSTRO_SOURCE_MIRROR_URL in local.conf), this way force fetched from the upstream, which resolved the mismatch issue. Hence reverting my commit.

But i am not sure, how it will be resolved at CI environment.